code/lib/cli/src/js-package-manager/JsPackageManager.ts

```diff
@@ -484,13 +478,19 @@ export abstract class JsPackageManager {
         stdio: stdio ?? 'pipe',
         encoding: 'utf-8',
         shell: true,
+        cleanup: true,
         env,
         ...execaOptions,
       });
 
       return commandResult.stdout ?? '';
     } catch (err) {
-      if (ignoreError !== true) {
+      if (
```

Comment:
Can we do the signal check here as well?

Reply:
Above we removed the try/catch block (which was wrong, since we removed the functionality of setting `ignoreError !== true). So I would just remove the whole part `err.killed !== true && err.isCanceled !== true && !err.message?.includes('Command was killed with SIGINT')` here as well

Reply:
https://github.com/storybookjs/storybook/pull/23538/commits/2d3e76a54400f3cffbc86f0117efc539228d7e67